app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,75 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+body {
+  font-family: sans-serif;
+  background-color: whitesmoke;
+}
+section {
+  background-color: white;
+  width: 500px;
+  margin: auto;
+  border: 2px solid black;
+  padding: 20px;
+  padding-right: 48px;
+  display: grid;
+  grid-template-columns: 1fr 3fr;
+  grid-column-gap: 20px;
+
+}
+
+section h1 {
+  font-size: 25px;
+  text-transform: uppercase;
+}
+a {
+  font-size: 12px;
+  text-decoration: none;
+  color: white;
```

Comment:
Because of this style, which makes all links turn white, it was impossible for me to see the link to Mark Complete or Mark Incomplete, so I almost thought that this feature wasn't done! Watch out with your styles and think about what the user sees.